h2o-hadoop/h2o-mapreduce-generic/src/main/java/water/hadoop/h2odriver.java

```diff
@@ -1369,6 +1410,11 @@ private int run2(String[] args) throws Exception {
       addMapperArg(conf, s);
     }
 
+    if (client) {
+      addMapperArg(conf, "-md5skip");
```

Comment:
hmm, is that necessary in this case? Unless the client accidentally connects to another cluster we don't need a version check (it is the same compiled code).

In my point of view the only thing that needs to be checked is if I am talking to the right cluster.